migrate/livesync-for-s3.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Livesync S3 to Timescale Cloud
+excerpt: Synchronize data from S3 to Timescale Cloud service in real time
+products: [cloud]
+keywords: [migration, low-downtime, backup]
+tags: [recovery, logical backup, replication]
+---
+
+import PrereqCloud from "versionContent/_partials/_prereqs-cloud-only.mdx";
+import EarlyAccessNoRelease from "versionContent/_partials/_early_access.mdx";
+
+# Livesync from S3 to Timescale Cloud
+
+You use $LIVESYNC to synchronize all the data, or specific tables, from an S3 bucket to your
+$SERVICE_LONG in real-time. You run $LIVESYNC continuously, turning S3 into a primary database with your
+$SERVICE_LONG as a logical replica. This enables you to leverage $CLOUD_LONG’s real-time analytics capabilities on
+your replica data.
+
+![$LIVESYNC_CAP view status](https://assets.timescale.com/docs/images/livesync-s3-view-status.png)
+
+You use $LIVESYNC for data synchronization, rather than migration. Livesync can:
+
+* Sync data from an S3 bucket instance to a $SERVICE_LONG:
+   - $LIVESYNC uses Glob patterns to identify the objects to sync.
+   - $LIVESYNC uses the objects returned for subsequent queries. This efficient approach means files are synced in
+    [lexicographical order][lex-order].
+   - $LIVESYNC watches an S3 bucket for new files and imports them automatically. $LIVESYNC runs on a configurable 
+     schedule and tracks processed files.
+   - To prevent system overload, $LIVESYNC track up to 100 files for each sync iteration. Additional checks only fill 
+      empty queue slots.
+   - For large backlogs, $LIVESYNC checks every minute until caught up. 
+
+* Sync data from multiple file formats:
+
+  * CSV: checked for compression, then processing using [timescaledb-parallel-copy][parallel-copy]
```

Comment:
I think we need to specify it is only `.gz` and `.zip` files.

Reply:
Done